### ISSUE TITLE
Replacetextsmart() cannot be used

### ISSUE THREAD

**Issue by niushuai1991** [ONLYOFFICE/DocumentBuilder#65]
**Describe your problem**:

I need to replace some text in the spreadsheet, but replacetextsmart () doesn't work properly.

*Be clear in your description of the problem.
Open an issue with a descriptive title and a summary in complete sentences.*

I used the sample code in the API documentation
https://api.onlyoffice.com/docbuilder/spreadsheetapi/api/replacetextsmart

Operation results:

```
execute_run_code: oRange.Select();
execute_run: Uncaught TypeError: oRange.Select is not a function
```

If I delete the code that contains the “oRange”，Operation results:
```
execute_run_code: Api.ReplaceTextSmart(["Cell 1", "Cell 2"]);
execute_run: Uncaught TypeError: Api.ReplaceTextSmart is not a function
```

**Steps to reproduce the problem**:
1. copy code from https://api.onlyoffice.com/docbuilder/spreadsheetapi/api/replacetextsmart
2. save the code to the file demo.docbuilder
3. excute command "docbuilder.exe  demo.docbuilder"

**DocumentBuilder version**:
documentbuilder-4.4.1

**Installation method**:
Extract and use

**Operating system**:
Windows 10 20H2

**Comment by ShockwaveNN** [comment]
Hi

> documentbuilder-4.4.1

This version is very old, could you update it to the latest stable version?

**Comment by niushuai1991** [comment]
Yes，I need an updated version，I downloaded the latest stable version from here
https://www.onlyoffice.com/zh/download.aspx#builder
But it still doesn't work,
“docbuilder.exe demo.docbuilder” the result of the command is：
Uncaught TypeError: this.Uy.zta is not a function


I think the links in this document need to be updated.
https://api.onlyoffice.com/docbuilder/gettingstarted

**Comment by ShockwaveNN** [comment]
> Uncaught TypeError: this.Uy.zta is not a function

Yeah, thanks, I think you're right, I've got the same error on v6.4.0, I'll research this right now

**Comment by ShockwaveNN** [comment]
I've localized the problem - problem not in `ReplaceTextSmart` method, but in `Range#select()`

This example also failing:
https://api.onlyoffice.com/docbuilder/spreadsheetapi/apirange/select

I've created bug 52684 in our private issue tracker

**Comment by niushuai1991** [comment]
Thanks for your answer.
The task I am doing is to find the entire spreadsheet and replace specific text with what I want, like "replace all". I want to know if I can finish my work before this bug is fixed?

**Comment by ShockwaveNN** [comment]
Sorry, I'm not sure that there is a workaround but we hope to include a fix of your problem in the next release v6.4.1 which we hope will be available next week if we do not find any other major problems

**Comment by niushuai1991** [comment]
Thank you, I'm waiting for your good news.

**Comment by niushuai1991** [comment]
Thank you. It's a great job. I'm waiting for your good news.

**Comment by ShockwaveNN** [comment]
Our dev team fixed it in https://github.com/ONLYOFFICE/sdkjs/commit/70ed6453bc392d0ed014d47d3e6d210a24dded8a

And we really hope that we can release this version next week (but no promises)

**Comment by ShockwaveNN** [comment]
DocumentBuilder v6.4.1 is released and this issue is fixed
